### PR TITLE
Protect: Add/empty status

### PR DESCRIPTION
**Pull request by leogermani**
#### Changes proposed in this Pull Request:

* Creates the empty status, when the plugin does not have any information yet.

#### Jetpack product discussion

N/A

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

* Connect and go to Protect
* Change your response_status to empty in your wp-config:
```
define( 'JETPACK_PROTECT_DEV__BYPASS_CACHE', true );
define( 'JETPACK_PROTECT_DEV__API_RESPONSE_TYPE', 'empty' );
```
* Refresh the page and see the new screen

Note 1: The original layout has a progress bar we will not implement - this will still need design review so the top margin in the Header is a temporary fix
Note 2: There are some issues in small viewports that will be handled in another PR. Same thing for the Cols with grid value zero. This is also being mistakenly used in the Footer and will be fixed later

Captura de tela de 2022-04-28 19-07-17

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202183900595806

### CODE REVIEW

File: projects/plugins/protect/src/js/components/admin-page/index.jsx [hunk 1]
```diff
@@ -47,6 +57,35 @@ const Admin = () => {
...
+	 * When there's no information yet. Usually when the plugin was just activated
+	 */
+
+	if ( ! lastChecked ) {
+		return (
+			<AdminPage moduleName={ __( 'Jetpack Protect', 'jetpack-protect' ) }>
+				<AdminSectionHero>
+					<Container horizontalSpacing={ 3 } horizontalGap={ 7 }>
+						<Col sm={ 4 } md={ 4 } lg={ 6 }>
+							<H3 mt={ 8 }>{ __( 'Your results will be ready soon', 'jetpack-protect' ) }</H3>
```
Comment (retrofox) [inline projects/plugins/protect/src/js/components/admin-page/index.jsx:70]: I'm not sure about adding the mt here. I think it should be addressed at layout level

**Review by retrofox — COMMENTED** [review]
Depending on the viewport size, the image gets cropped.

image

**Comment by leogermani** [comment]
> Depending on the viewport size, the image gets cropped.

Known issue, see "note 2" on the testing instructions

**Review by retrofox — APPROVED** [review]
I think we should pay a little bit more attention to spacing, position, etc. Also, I think It could/should be implemented with the Dialog, or something similar, component.
Happy to continue in a follow-up PR

File: projects/plugins/protect/src/js/components/admin-page/index.jsx (hunk 1 above)
Comment (leogermani) [reply projects/plugins/protect/src/js/components/admin-page/index.jsx:70]: Known issue: See note 1 in the test instructions :)

**Pull request merged**